browser/brave_content_browser_client.cc

```diff
@@ -604,6 +606,11 @@ void BraveContentBrowserClient::RegisterBrowserInterfaceBindersForFrame(
   }
 #endif
 
+#if BUILDFLAG(ENABLE_SPEEDREADER)
+  chrome::internal::RegisterWebUIControllerInterfaceBinder<
```

Comment:
Is this needed only for PanelV2 feature flag?

Reply:
good catch, needs a predicate on feature flag here